Mathlib/RingTheory/Valuation/Integers.lean

```diff
@@ -76,6 +76,14 @@ theorem one_of_isUnit {x : O} (hx : IsUnit x) : v (algebraMap O R x) = 1 :=
     exact mul_le_mul_left' (hv.2 (u⁻¹ : Units O)) _
 #align valuation.integers.one_of_is_unit Valuation.Integers.one_of_isUnit
 
+theorem one_of_isUnit' {x : O} (hx : IsUnit x) (H : ∀ x, v (algebraMap O R x) ≤ 1) :
```

Comment:
Am I correct that this does not use `(hv : Integers v O)`? Since it is not immediate to see what hypotheses are included or not, maybe use this to golf `one_of_isUnit` or add a docstring or move it out of the `Integers` namespace instead to make It clearer?

Reply:
You are certainly right, but since this is never needed when you #check it the variable `hv` does not show up.